pkg/devserver/discovery.go

```diff
@@ -0,0 +1,105 @@
+package devserver
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"net/http"
+	"strconv"
+	"sync"
+	"time"
+)
+
+var (
+	// Ports indicate the default ports that we attempt to scan on localhost
+	// when discovering SDK-based endpoints
+	Ports = []int{80, 3000, 3001, 3002, 3003, 3004, 3005, 5000, 8000, 8080, 8081, 8888}
```

Comment:
If my process was running on a port not specified here, what would be my action?

Either we could add the ability to specify extra ports to scan, or we could put the onus on the SDK UI to find the dev server and register itself?

Reply:
I think @djfarrelly had the same question and recommended the PORT env var, which we've added.  This will add your PORT to this list for autodiscovery.